mmcv/ops/nms.py

```diff
@@ -90,7 +104,7 @@ def symbolic(g, boxes, scores, iou_threshold, sigma, min_score, method,
 
 
 @deprecated_api_warning({'iou_thr': 'iou_threshold'})
-def nms(boxes, scores, iou_threshold, offset=0):
+def nms(boxes, scores, iou_threshold, score_threshold=0, max_num=-1, offset=0):
```

Comment:
Could update docstring for argument `score_threshold` and `max_num`.

Reply:
Done.